src/js/main.js

```diff
@@ -41,18 +50,50 @@ class Timeline extends React.Component {
 		}
 		else {
 			homeTimelineResultOutput = renderedTimeline(this.state.homeTimeline);
-			homeTimelineResultClass = "successGetTimeline";			
+			homeTimelineResultClass = "successGetTimeline";
+		}
+	}
+
+	userTimelineResult() {
+		if(this.state.userTimelineError) {
+			userTimelineResultOutput = "There was a problem on the server side, please try again later";
+			userTimelineResultClass = "error";	
 		}
-		return React.createElement('div', {className: 'Timeline'}, 
-				[React.createElement('div', {className: 'buttonContainer', key: 'buttonCont'}, 
-					React.createElement(HomeTimelineButton, {className: 'timelineButton', onClickFunc: () => getHomeTimeline(this.homeTimelineCallback)}, 'Get Timeline')),
-				React.createElement(TimelineResultComp, {className: homeTimelineResultClass, key: 'timelineResComp', timelineResult: homeTimelineResultOutput}, null)]);
+		else if(_.isNull(this.state.homeTimeline) && !this.state.userTimelineError) {
+			userTimelineResultOutput = " ";
+			userTimelineResultClass = "pending";	
+		}
+		else {
+			userTimelineResultOutput = renderedTimeline(this.state.userTimeline);
+			userTimelineResultClass = "successGetTimeline";			
+		}
+	}
+
+	componentDidMount() {
+		getHomeTimeline(this.homeTimelineCallback);
+		getUserTimeline(this.userTimelineCallback);
+	}
+
+	render(){
+		{this.homeTimelineResult()};
+		{this.userTimelineResult()};
+		return React.createElement('div', {className: 'Timeline'},
+					[React.createElement('div', {className: 'homeTimelineContainer', key: 'homeTimelineCont'}, 
```

Comment:
for the future you can define your own function that stands in for React.createElement so your code is less verbose